### PR TITLE
README makeover

### PR DESCRIPTION
**Pull request by grantjoy**
Rewriting the readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,11 +95,11 @@ AWS.IAM.MFAEnabled
 [INFO]: <current working directory>/tmp/panther-analysis-2020-03-23T12-48-18.zip
 ```
 
-### Upload packages to Panther directly. 
+Or upload packages directly into Panther:
 
 _Note, this expects your environment to be setup the same way as if you were using the AWS CLI, see the setup instructions [here](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html). We also recommend using a credentials manager such as [aws-vault](https://github.com/99designs/aws-vault)._
```
Comment (grantjoy) [inline README.md:100]: Is this still true? Can't we use API keys OR AWS creds?

**Review by nskobov — APPROVED** [review]
looks good to me. for other reviewers i found it helpful to look at the rendered view instead of the text diff:
https://github.com/panther-labs/panther_analysis_tool/blob/d617a4f0a20d22804e2f292138c2343379ce59d7/README.md

File: README.md (hunk 1 above)
Comment (darwayne) [reply README.md:100]: correct users can utilize an api key OR aws creds, probably be good to call that out I don't think we ever updated readme to reflect that

**Review by darwayne — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (grantjoy) [reply README.md]: Thanks @darwayne can you stamp again? I removed that line.

File: README.md [hunk 2]
```diff
@@ -125,20 +127,20 @@ AWS.CloudTrail.MFAEnabled
   "totalRules": 1
 }
 ```
-## Filtering
+### Filtering
 
-The `test`, `zip`, and `upload` commands all support filtering. Filtering works by passing the `--filter` argument with a list of filters specified in the format `KEY=VALUE1,VALUE2`. The keys can be any valid field in a policy or rule. When using a filter, only anaylsis that matches each filter specified will be considered. For example, the following command will test only items with the AnalysisType of policy AND the severity of High:
+The `test`, `zip`, and `upload` commands all support filtering. Filtering works by passing the `--filter` argument with a list of filters specified in the format `KEY=VALUE1,VALUE2`. The keys can be any valid field in a policy or rule. When using a filter, only anaylsis that matches each filter specified will be considered. For example, the following command will test only items with the AnalysisType as `policy` AND severity as `High``:
```
Comment (grantjoy) [inline README.md:132]:
```suggestion
The `test`, `zip`, and `upload` commands all support filtering. Filtering works by passing the `--filter` argument with a list of filters specified in the format `KEY=VALUE1,VALUE2`. The keys can be any valid field in a policy or rule. When using a filter, only anaylsis that matches each filter specified will be considered. For example, the following command will test only items with the AnalysisType as `policy` AND severity as `High`:
```

File: README.md [hunk 3]
```diff
@@ -125,20 +127,20 @@ AWS.CloudTrail.MFAEnabled
...
+$ panther_analysis_tool test --path tests/fixtures/valid_policies --filter AnalysisType=policy Severity=High
 [INFO]: Testing analysis packs in tests/fixtures/valid_policies
 
 AWS.IAM.BetaTest
 	[PASS] Root MFA not enabled fails compliance
 	[PASS] User MFA not enabled fails compliance
 ```
 
-Whereas the following command will test items with the AnalysisType policy OR rule, AND the severity High:
+Alternately, the following command will test items with the AnalysisType `policy` OR `rule``, AND the severity `High``:
```
Comment (grantjoy) [inline README.md:143]:
```suggestion
Alternately, the following command will test items with the AnalysisType `policy` OR `rule`, AND the severity `High`:
```

File: README.md [hunk 4]
```diff
@@ -217,48 +219,55 @@ Failed Tests Summary
...
+From your (panther_analysis)[https://github.com/panther-labs/panther-analysis] content repository, run:
+```shell
 pipenv install --editable ../relative/path/to/panther_analysis_tool
 ```
 
-Then you can test your changes to PAT by running 
-```
+Then test changes to PAT:
+```shell
 pipenv run panther_analysis_tool <cmd>
```
Comment (grantjoy) [inline README.md:271]:
```suggestion
pipenv run panther_analysis_tool test
```

**Review by nskobov — APPROVED** [review]

**Pull request merged**